start_esp/start_esp.py

```diff
@@ -569,6 +571,17 @@ def make_argparser():
         action='store_true',
         help=argparse.SUPPRESS)
 
+    parser.add_argument('--client_real_ip_header', default=None, help='''
+    Defines the request header field whose value will be used to be used as
+    the real client address.
```

Comment:
I don't think we want to extract from any http header.   I don't live to introduce two flags either.
it will be specific to XFF.   something like xff_position_for_client_ip

Reply:
Applied real_client_ip_from_header_config message in server config